logbook-core/src/main/java/org/zalando/logbook/DefaultHttpLogFormatter.java

```diff
@@ -59,19 +58,19 @@ private String formatStatusLine(final HttpResponse response) {
     private <H extends HttpMessage> String format(final H message, final String type, final String correlationId,
             final Function<H, String> lineCreator)
             throws IOException {
-        final List<String> lines = Lists.newArrayListWithExpectedSize(4);
+        final List<String> lines = new ArrayList<>();
```

Comment:
Could still give the expected capacity (which should be 5 instead of 4 to avoid resizing). Indeed for performance reasons this should use a StringBuilder which could be passed to the helper methods. Could be a separate PR though.